### ISSUE TITLE
sp_Blitz: add new check for backups to NUL

### ISSUE THREAD

**Issue by DBAdminDB** [BrentOzarULTD/SQL-Server-First-Responder-Kit#2640]
For some third-party backup programs (like Veeam) there is an option to truncate logs, which take log backup to NUL,
so if there is an existing maintenance plan on the server, those NUL backups could break the log backup chain.

my suggestion is to add a check to sp_blitz which checks whether there are backups to NUL or not.

**Comment by BrentOzar** [comment]
Hi, and welcome to the project. When you opened this issue, there was a template, and it looks like you overwrote that. One of the questions said:

**Are you ready to build the code for the feature?**
As much as we'd love to build everything that everyone wants for free, we need your help. Open source is built with your help and code. Are you ready to commit time to this project? Have you got existing code you can help contribute to solve the problem?

Can you answer that one? Thanks!

**Comment by DBAdminDB** [comment]
I guess I can.

do you have an idea how to approach it (relevant temp tables and so on...)

where should I post the code for you to check?

**Comment by BrentOzar** [comment]
Sure, on the main page of the repo here, scroll down and there's a lot of documentation on how to contribute:

https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/tree/main

Then, when you want to learn how sp_Blitz works, start here:

https://www.brentozar.com/archive/2017/02/tourists-guide-sp_blitz-source-code-part-1-big-picture/

**Comment by DBAdminDB** [comment]
What link can I use as a URL for this issue?

**Comment by BrentOzar** [comment]
Sure, use this: https://www.brentozar.com/archive/2009/08/backup-log-with-truncate-only-in-sql-server-2008/

**Comment by DBAdminDB** [comment]
ok, I've added it as a check with the ID 256.

it was a little bit hard to understand how to add a check, I hope I did it ok...
I'll create a pull request in a min.

**Comment by BrentOzar** [comment]
Thanks for the pull request! Looks good, merged into dev, will be in the November release with credit to you in the release notes.

**Comment by hjvanrooyen** [comment]
Hallo everyone

When I ran sp_Blitz, I see that there are finding enties on “Log Backups to NUL”.

I make use of Veeam for server backups and I also have maintenance plan’s that backups SQL as well. I also noticed that in the Veeam backup output that Veeam truncating transaction logs.

Is it possible to tell what is the best practice here?

Regards

![image](https://user-images.githubusercontent.com/82569667/114850841-78833800-9de1-11eb-82ba-dd2f0857f740.png)

![image](https://user-images.githubusercontent.com/82569667/114850935-981a6080-9de1-11eb-9cc8-e1090978eab5.png)
